src/Symfony/Component/Profiler/ProfileData/ConfigData.php

```diff
@@ -0,0 +1,185 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Profiler\ProfileData;
+
+/**
+ * Class ConfigData
+ *
+ * @author Jelte Steijaert <jelte@khepri.be>
+ */
+class ConfigData extends AbstractProfileData
+{
+    public function getApplicationName()
```

Comment:
don't forget phpdocs here as this is part of public api, same for ``getApplicationVersion``